src/overview/copy-paster/background/storage.test.ts

```diff
@@ -0,0 +1,92 @@
+import StorageManager from '@worldbrain/storex'
+import { registerModuleMapCollections } from '@worldbrain/storex-pattern-modules'
+import { setupBackgroundIntegrationTest } from 'src/tests/background-integration-tests'
+
+import CopyPasterBackground from './'
+
+async function setupTest() {
+    const {
+        backgroundModules,
+        storageManager,
+    } = await setupBackgroundIntegrationTest()
+
+    const copyPaster: CopyPasterBackground = backgroundModules.copyPaster
+
+    registerModuleMapCollections(storageManager.registry, {
+        copyPaster: copyPaster.storage,
+    })
+
+    return { copyPaster }
+}
+
+describe('create template', () => {
+    test('should be able to create template', async () => {
```

Comment:
Nitpick: don't forget articles, also in other tests, like 'should be able to create **A** template' or 'create templates'. I know articles are not there in Slavic languages (I speak Czech myself  :)  )